### ISSUE TITLE
Docs teams needs some feedback on compiler switches in VB

### ISSUE THREAD

**Issue by KathleenDollard** [dotnet/vblang#267]
This is to let folks here know about an issue you might overlook.

Weigh in here: https://github.com/dotnet/docs/issues/4450

**Comment by tverweij** [comment]
I vote for -.
But can't whe have them both?

**Comment by KathleenDollard** [comment]
Sorry if there was any lack of clarity on this. 

- and / will both continue to work for compiler switches. Changing that would be a backward compatibility break and would not be done. 

The documentation needs to display one or the other, along with a note that they both work. The question is only which should be displayed in the documentation.

**Comment by AdamSpeight2008** [comment]
I'd personally go with `-` as it applicable to C#, as `/` can be pain to use especially is needs to be escaped.

**Comment by KathleenDollard** [comment]
@AdamSpeight2008 Can you weigh in at the docs link so Maira can count your vote.

**Comment by franzalex** [comment]
Thanks for bringing this up @KathleenDollard. I'd have missed it otherwise.

By the way, do votes from non-Microsoft persons count?

**Comment by reduckted** [comment]
@franzalex of course they count. If they didn't, what would be the point in asking on GitHub? :smile:

**Comment by KathleenDollard** [comment]
@franzalex 

Yes, definitely. In fact in this and many cases ONLY non-Microsoft persons count.
